src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -442,3 +453,63 @@ This can happen for a couple of reasons:
 	}
 	return err
 }
+
+func NodeAddr(port uint) string {
+	return fmt.Sprintf(
+		"127.0.0.1:%d",
+		port,
+	)
+}
+
+func RenderClusterInfoStatus(c Client) error {
+	nodes, err := GetExistingNodes(c)
```

Comment:
I would expect this entire function to live in status.go, especially since status_test.go tests specifically this function

Reply:
Specifically, I think it'd be `GetExistingNodes; checks(); renderClusterInfo(...);`, using the `renderClusterInfo` that already lives in `start.go`